assets/js/hooks/file_manager.js

```diff
@@ -0,0 +1,18 @@
+const FileManager = {
+  mounted() {
+    this.handleEvent("remove_file", ({ id }) => {
+      document.getElementById(id)?.remove();
```

Comment:
It's not necessary to manually remove the element from the DOM. Phoenix LiveView takes care of that automatically.